GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+
+## Changes to the Governance
+
+Changes to this Governance may be proposed via a Pull Request on the Governance itself.  
+Amendments are accepted with 2/3rd majority consent of the committee as per the [Decision Process](#decision-process) outlined above.
+
+Proposals and amendments to the charter are available for at least a period of one week for comments and questions before a vote will occur.
+
+## Authority, Facilitation, and Decision Making
+
+Ideally, most decisions will be made at the lowest possible level within the project: within individual working groups. When this is not possible, the Steering Committee can help facilitate a conversation to work through the contended issue. When facilitation by the Steering Committee does not resolve the contention, the Steering Committee may have to decide.
+
+For example, Technical governance is expected to be performed by the Maintainers of the various project areas and subprojects, as defined in the appropriate OWNERS / MAINTAINERS files.
+
+Note that if the committee is called to resolve contended decisions regularly, it is a symptom of a larger problem in the community that will need to be addressed.
+
+## Inactivity
+
+It is important for members to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a loss of trust in the project.
+
+* Inactivity is measured by
+  * Periods of no contributions for longer than 6 months
+  * Not having 20 contributions in the preceding 12-month window for more than 6 months
+* Consequences of being inactive include
+  * Involuntary removal or demotion
+  * Being asked to move to Emeritus status
+  * Not counting against quorum in votes
+
+## Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when responsibilities and requirements aren't being met. This may include repeated patterns of inactivity, extended periods of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opening up opportunities for new contributors to step in.
+
+Inactive members can be automatically removed.
+
+Involuntary removal or demotion of Maintainers or Steering Committee members is handled through a vote by a majority of the current Steering Committee. If this pertains to a Steering Committee member, that member is not eligible to discuss or vote on the matter.
+
+## Stepping Down/Emeritus Process
+
+If and when contributors' commitment levels change, contributors can consider stepping down (moving down the contributor ladder) vs moving to emeritus status (completely stepping away from the project).
```

Comment:
This lower section seems to talk about the roles beyond the Steering Committee. For me, reading this was confusing because most of this document is focused solely on the Steering Committee. I think this section might make more sense with the roles instead of here.